docs/index.rst

```diff
@@ -76,32 +76,77 @@ API <https://developer.spotify.com/documentation/web-api/>`_ documentation.
 Installation
 ============
 
-Install or upgrade *Spotipy* with::
+**Installing Spotipy on Windows**:
+
+- Install Python: Download and install Python from the official Python website (https://www.python.org/downloads/).
+
+- Install Spotipy: Open a command prompt and run pip install spotipy.
+Open command prompt and run the following::
+
+   pip install spotipy
+
+- Authenticate your app: Register your app on the Spotify Developer Dashboard and obtain a client ID and client secret.
+
+**Installing Spotipy on macOS**:
+
+
+- Install Homebrew: Install Homebrew, a package manager for macOS.
+
+- Run the followng command::
+
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+- Install Python: Run brew install python in the terminal::
+
+   brew install python
+
```

Comment:
I appreciate beginners can use this package, but any python library shouldn't have to explain how to install python, it's assumed that it's a prerequisite